bookkeeper-benchmark/src/test/java/org/apache/bookkeeper/benchmark/TestBenchmark.java

```diff
@@ -100,14 +108,7 @@ public void run() {
                 bk.close();
             }
         }
-        for (int i = 0; i < 60; i++) {
-            if (!t.isAlive()) {
-                break;
-            }
-            Thread.sleep(100);
-        }
-
-        Assert.assertFalse("Thread should be finished", t.isAlive());
+        Assert.assertEquals(Integer.valueOf(0), f.get(30, TimeUnit.SECONDS));
```

Comment:
can we get rid of timed wait here? we should just rely on the unit test framework to timeout the test case.

Reply:
changed.